File: 01-Assignment-02-02-2026/Using_C/program_7.c
/* Program 7: Check whether the given integer is a multiple of 5 */

#include <stdio.h>

int main() {
    int num ;
    printf("Enter Your Number : ");
    scanf("%d",&num);

    if (num%5 == 0){
        printf("yes,It is mutiple of 5");
    }
    else{
        printf("No, It is not Multiple of 5");
    }
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program7.java
/*
Program 7
Java program to check whether the given integer is a multiple of 5
*/

import java.util.Scanner;

public class Program7 {

    public static void main(String[] args) {
        
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number : ");
        int number = scanner.nextInt();

        if (number % 5 == 0){
            System.out.println("Yes " + number + " is multiple of 5.");
        }
        else{
            System.out.println("No " + number + " is not a multiple of 5.");

        }

        scanner.close();
    }

}
